usercommandmgmt/configurable.py

```diff
@@ -1,12 +1,14 @@
-#contains all the configurable fields for this cog
+# contains all the configurable fields for this cog
 
-#set to false to disable the pre-command creation moderator approval system
-command_moderation = False
+# specifies the number of commands granted to specific roles
+role_cmd_limits = {"Tier 1 Simp": 1, "Tier 2 Simp": 3, "Phillip Paik himself": 5}
 
-#specifies the number of commands granted to specific roles
-role_cmd_limits = {
-    "Tier 1 Simp" : 1,
-    "Tier 2 Simp": 3,
-    "Phillip Paik himself": 5
-}
+# admin command management
+# set to false to disable the pre-command creation moderator approval system
+command_moderation = True
 
+# the name of the channel used for moderation
+mod_channel_name = "usercommandmgmt-admin-approval"
```

Comment:
How about adding a command that sets whatever channel the command is used in to the moderation channel? 

You can see an example of how to do this in the default Report cog's command here: https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/redbot/cogs/reports/reports.py#L108